src/Shared/AssemblyFolders/AssemblyFoldersEx.cs

```diff
@@ -483,6 +488,11 @@ IEnumerator IEnumerable.GetEnumerator()
         {
             return ((IEnumerable<AssemblyFoldersExInfo>)this).GetEnumerator();
         }
+
+        internal IEnumerable<string> UniqueDirectoryPaths()
```

Comment:
nit: Could be a property instead of method. (Sorry, my fault, I misled you with me previous comment.)

Reply:
Fixed, should be OK now.